src/app/categories/bookmarks/bookmarks.js

```diff
@@ -17,7 +17,8 @@ angular.module('categories.bookmarks', [
             })
         ;
     })
-    .controller('BookmarksCtrl', function ($scope,$stateParams  ) {
-        $scope.currentCategoryName = $stateParams.category;
+    .controller('BookmarksCtrl', function ($stateParams) {
```

Comment:
main should be named bookmark here because it is the bookmark controller, not main